test/test_utils/ec2.py

```diff
@@ -361,9 +361,10 @@ def execute_ec2_training_test(connection, ecr_uri, test_cmd, region=DEFAULT_REGI
     connection.run(f"$(aws ecr get-login --no-include-email --region {region})", hide=True)
 
     # Run training command
+    shm_setting = '--shm-size="1g"' if large_shm else ""
```

Comment:
nit: should we keep it as gb instead of g ?

Reply:
Not sure if "gb" is an accepted and standard syntax.
Following the regulation on this page (one line ahead of this link line) https://docs.docker.com/engine/reference/run/#user-memory-constraints